docs/io/ascii/read.rst

```diff
@@ -23,6 +23,7 @@ format, for example::
    >>> data = astropy.io.ascii.read('t/nls1_stackinfo.dbout', data_start=2, delimiter='|')  # doctest: +SKIP
    >>> data = astropy.io.ascii.read('t/simple.txt', quotechar="'")  # doctest: +SKIP
    >>> data = astropy.io.ascii.read('t/simple4.txt', format='no_header', delimiter='|')  # doctest: +SKIP
+   >>> data = astropy.io.ascii.read('t/tab_and_space.txt', delimiter='\s')  # doctest: +SKIP
```

Comment:
The paths also contain deprecated sequences but these aren't platform-independant anyway so that's not really surprising.  

Especially the one with `'t\nl ....'` may even break in very funny ways (because it contains a newline `\n`).